tests/integration/commands/blobs/blobs-set.test.ts

```diff
@@ -0,0 +1,284 @@
+          expect(log).toHaveBeenCalledWith(successMessage)
+        })
+      })
+
+      test('should log error message if adding a key fails', async () => {
+        await withMockApi(routes, async ({ apiUrl }) => {
+          Object.assign(process.env, getEnvironmentVariables({ apiUrl }))
+
+          const mockSet = vi.fn().mockRejectedValue('')
+          vi.mocked(reportError).mockResolvedValue()
+          ;(getStore as any).mockReturnValue({
+            set: mockSet,
+          })
+
+          const promptSpy = spyOnMockPrompt()
+
+          try {
+            await runMockProgram(['', '', 'blobs:set', storeName, key, newValue, '--force'])
+          } catch (error) {
+            expect(error.message).toContain(
+              `Could not set blob ${chalk.yellow(key)} in store ${chalk.yellow(storeName)}`,
+            )
+          }
+
+          expect(promptSpy).not.toHaveBeenCalled()
+
+          expect(log).not.toHaveBeenCalledWith(warningMessage)
+          expect(log).not.toHaveBeenCalledWith(overwriteNotice)
+          expect(log).not.toHaveBeenCalledWith(successMessage)
+        })
+      })
+    })
+
+    describe('prompts should not show in a non-interactive shell or in a ci/cd enviroment', () => {
+      test('should not show prompt in an non-interactive shell', async () => {
+        setTTYMode(false)
+
+        await withMockApi(routes, async ({ apiUrl }) => {
+          Object.assign(process.env, getEnvironmentVariables({ apiUrl }))
+
+          // Mocking the store.get method to return a value (simulating that the key already exists)
+          const mockGet = vi.fn().mockResolvedValue('my-value')
+          const mockSet = vi.fn().mockResolvedValue('true')
+
+          ;(getStore as any).mockReturnValue({
+            get: mockGet,
+            set: mockSet,
+          })
+
+          const promptSpy = spyOnMockPrompt()
+
+          await runMockProgram(['', '', 'blobs:set', storeName, key, newValue, '--force'])
+
+          expect(promptSpy).not.toHaveBeenCalled()
+
+          expect(log).not.toHaveBeenCalledWith(warningMessage)
+          expect(log).not.toHaveBeenCalledWith(overwriteNotice)
+          expect(log).toHaveBeenCalledWith(successMessage)
+        })
+      })
+
+      test('should not show prompt in a ci/cd environment', async () => {
+        setCI(true)
+        await withMockApi(routes, async ({ apiUrl }) => {
+          Object.assign(process.env, getEnvironmentVariables({ apiUrl }))
+
+          // Mocking the store.get method to return a value (simulating that the key already exists)
+          const mockGet = vi.fn().mockResolvedValue('my-value')
+          const mockSet = vi.fn().mockResolvedValue('true')
+
+          ;(getStore as any).mockReturnValue({
+            get: mockGet,
+            set: mockSet,
+          })
+
+          const promptSpy = spyOnMockPrompt()
+
+          await runMockProgram(['', '', 'blobs:set', storeName, key, newValue, '--force'])
```

Comment:
I might be misunderstanding but don't we want to not include the `--force` arg here so that we test is it gets set automatically?
```suggestion
          await runMockProgram(['', '', 'blobs:set', storeName, key, newValue])
```

Reply:
Done.